uaclient/cli.py

```diff
@@ -1290,8 +1291,10 @@ def action_auto_attach(args, *, cfg):
 
     try:
         actions.auto_attach(cfg, instance)
-    except exceptions.UrlError:
-        event.info(messages.ATTACH_FAILURE)
+    except (exceptions.UrlError, HTTPError) as e:
```

Comment:
I am not sure, if we should separate the errors. Did this for the message `ATTACHED_FAILURE`.